skbeam/io/tests/test_fit2d_save.py

```diff
@@ -44,17 +44,24 @@
 import numpy as np
 from skbeam.io.fit2d import fit2d_save, read_fit2d_msk
 from numpy.testing import assert_array_equal
-import pytest
+import sys
+
+if sys.version_info >= (3, 0):
+    from tempfile import TemporaryDirectory
+else:
+    from backports.tempfile import TemporaryDirectory
+
 
-@pytest.mark.xfail(raises=IOError)
 def test_save_output_fit2d():
```

Comment:
I was wrong. pytest keeps the last 3 runs. [From the docs](https://docs.pytest.org/en/latest/tmpdir.html):

> Temporary directories are by default created as sub-directories of the system temporary directory. The base name will be pytest-NUM where NUM will be incremented with each test run. Moreover, entries older than 3 temporary directories will be removed.